File: RetrofitSample/app/src/main/java/com/example/retrofitsample/GithubService.java
package com.example.retrofitsample;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;
import retrofit2.http.Query;

public interface GithubService {

    @GET("/search/repositories")
    Call<GitHubResponse> listRepo(@Query("q") String user);
}

File: WorkManager/app/src/main/java/com/example/workmanager/GitHubService.kt
package com.example.workmanager

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

/**
Created by sanjana on 30/4/20
 */
interface GitHubService {
    @GET("/search/repositories")
    fun listRepo(@Query("q") user: String?): Call<GitHubResponse?>?
}